--- envvars     2018-06-17 03:38:17.898222002 +0000
+++ /etc/apache2/envvars        2018-06-17 03:40:25.429956000 +0000
@@ -16,11 +16,11 @@ fi
 export APACHE_RUN_USER=www-data
 export APACHE_RUN_GROUP=www-data
 # temporary state file location. This might be changed to /run in Wheezy+1
-export APACHE_PID_FILE=/var/run/apache2/apache2$SUFFIX.pid
-export APACHE_RUN_DIR=/var/run/apache2$SUFFIX
-export APACHE_LOCK_DIR=/var/lock/apache2$SUFFIX
+export APACHE_PID_FILE=/home/vagrant/var/run/apache2/apache2$SUFFIX.pid
+export APACHE_RUN_DIR=/home/vagrant/var/run/apache2$SUFFIX
+export APACHE_LOCK_DIR=/home/vagrant/var/lock/apache2$SUFFIX
 # Only /var/log/apache2 is handled by /etc/logrotate.d/apache2.
-export APACHE_LOG_DIR=/var/log/apache2$SUFFIX
+export APACHE_LOG_DIR=/home/vagrant/var/log/apache2$SUFFIX

 ## The locale used by some modules like mod_dav
 export LANG=C
